website/docs/components/stepper/indicator/partials/specifications/states.md

```diff
@@ -0,0 +1,9 @@
+## States
+
+### Step indicator
+
+![Step indicator states](/assets/components/stepper/indicator/stepper-step-states.png =607x*)
```

Comment:
The images should be appropriately sized in Figma before export that you shouldn't need "=607x*"